packages/composite-checkout/demo/index.js

```diff
@@ -86,14 +86,23 @@ async function sendStripeTransaction( data ) {
 	};
 }
 
+async function applePayProcessor( data ) {
+	window.console.log( 'Processing apple-pay transaction with data', data );
```

Comment:
Is the use of `console.log` instead of `debug` intentional here? (It looks like it was here before this PR)

Reply:
nvm - I just realized this is in the demo :)